src/Main.elm

```diff
@@ -104,18 +104,36 @@ view model =
                 , ul [ class "fields__font-list" ]
                     <| List.indexedMap viewFont model
                 ]
-            , pre
-                [ class "main__font-faces" ]
-                [ List.filterMap viewFontFaces model
-                    |> String.join "\n"
-                    |> text
-                ]
+            , viewFontFaces model
+            ]
+        ]
+
+viewFontFaces : Model -> Html Msg
+viewFontFaces model =
+    let
+        selected =
+            List.filterMap pickSelected model
+    in
+        section [ class "font-faces" ]
+            [ viewFontsForPlatform "Web" (web selected)
+            , viewFontsForPlatform "Android" (android selected)
+            ]
+
+viewFontsForPlatform : String -> List String -> Html Msg
+viewFontsForPlatform heading fonts =
+    details []
+        [ summary [ class "font-faces__heading" ] [ text heading ]
+        , pre
+            [ class "font-faces__code" ]
+            [ fonts
+                |> String.join "\n"
+                |> text
             ]
         ]
 
-viewFontFaces : (Selected, Font) -> Maybe String
-viewFontFaces (isSelected, font) =
-    if isSelected then Just (fontFace font) else Nothing
+pickSelected : (Selected, Font) -> Maybe Font
```

Comment:
I'd avoid aliasing Boolean like this, but this is a comment for the wider app rather than this PR.

Reply:
What would you say the downsides of doing that are?

Reply:
If you want to take this approach I think you generally want to do something like the following instead:

```
type Selected
    = Yep
    | Nope
```
(with names that better fit your domain)

Or if you really want to wrap the type then even the following

```
type Selected = Constructor Bool
```
(again, with a good name)

With these two approaches you are actually getting type safety on your domain. By aliasing `Bool` to another name you create confusion through indirection (what is this `Selected` type?), and end up with something that is precisely identical to Boolean. With the alias you have now you can freely pass Booleans to Selecteds and vice versa, so you haven't really gained anything.

Using a `Bool` directly here would be fine, or the first example above is the best approach if you want to be explicit (and typesafe) that we are talking about `Selected` rather than `Bool`.

Reply:
The thing about using a `Bool` directly is that I find the model type becomes less helpful:

```elm
type alias Model =
    List (Bool, Font)
```

It's not really clear what the `Bool` is referring to. Unless you're suggested switch to `Bool` for the function definition but keep `Selected` for the model definition?

I think a custom type (i.e. the first example you gave) would also be my preferred approach in general 👍